src/topics/topicsReducer.js

```diff
@@ -42,6 +47,9 @@ export default (
 ): TopicsState => {
   switch (action.type) {
     case RESET_ACCOUNT_DATA:
+
+    // Reset to clear stale data; payload has no initial data for this model
+    case REGISTER_COMPLETE: // eslint-disable-line no-fallthrough
       return initialState;
```

Comment:
I think it's probably clearer to just have two cases here:
```suggestion
    case RESET_ACCOUNT_DATA:
      return initialState;

    case REGISTER_COMPLETE:
      // Reset to clear stale data; payload has no initial data for this model
      return initialState;
```

It's true that the implementation of the two cases is the same; but that one line isn't much code to duplicate.

And as the comment indicates, the reason why the two different action types lead to that result are different. So it's cleanest to think about them separately, and not try to make a common abstraction they share.